modules/media/php/media.class.inc

```diff
@@ -51,7 +51,7 @@ class Media extends \NDB_Menu_Filter
         // Set permission to access user-hidden files, currently based on superuser
         $this->hasHidePermission = $user->hasPermission("superuser");
 
-        return $user->hasPermission('media_read');
+        return $this->hasPermission('media_read') || $this->hasWritePermission;
```

Comment:
$this->hasPermission doesn't appear to be defined

Reply:
should be $user->hasPermission('media_read');

probably from my copy/pasted suggestion